specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2023-05-01/containerInstance.json

```diff
@@ -0,0 +1,2549 @@
+              "readOnly": true,
+              "description": "Current container instance state.",
+              "$ref": "#/definitions/ContainerState"
+            },
+            "previousState": {
+              "readOnly": true,
+              "description": "Previous container instance state.",
+              "$ref": "#/definitions/ContainerState"
+            },
+            "events": {
+              "readOnly": true,
+              "description": "The events of the container instance.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Event"
+              },
+              "x-ms-identifiers": [
+                "name",
+                "type"
+              ]
+            }
+          }
+        },
+        "resources": {
+          "description": "The resource requirements of the container instance.",
+          "$ref": "#/definitions/ResourceRequirements"
+        },
+        "volumeMounts": {
+          "description": "The volume mounts available to the container instance.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VolumeMount"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "livenessProbe": {
+          "description": "The liveness probe.",
+          "$ref": "#/definitions/ContainerProbe"
+        },
+        "readinessProbe": {
+          "description": "The readiness probe.",
+          "$ref": "#/definitions/ContainerProbe"
+        },
+        "securityContext": {
+          "description": "The container security properties.",
+          "$ref": "#/definitions/SecurityContextDefinition"
+        }
+      }
+    },
+    "SecurityContextDefinition": {
+      "description": "The security context for the container.",
+      "type": "object",
+      "properties": {
+        "privileged": {
+          "type": "boolean",
+          "description": "The flag to determine if the container permissions is elevated to Privileged."
+        },
+        "allowPrivilegeEscalation": {
+          "type": "boolean",
+          "description": "A boolean value indicating whether the init process can elevate its privileges"
+        },
+        "capabilities": {
+          "description": "The capabilities to add or drop from a container.",
+          "$ref": "#/definitions/SecurityContextCapabilitiesDefinition"
+        },
+        "runAsGroup": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Sets the User GID for the container."
+        },
+        "runAsUser": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Sets the User UID for the container."
+        },
+        "seccompProfile": {
+          "type": "string",
+          "description": "a base64 encoded string containing the contents of the JSON in the seccomp profile"
+        }
+      }
+    },
+    "SecurityContextCapabilitiesDefinition": {
```

Comment:
>SecurityContextCapabilitiesDefinition [](http://example.com/codeflow?start=5&length=37)

What will the capabilities look like? Can you also add a example for this?

Reply:
added Capabilities to example: [examples/ContainerGroupCreateConfidential.json](https://github.com/Azure/azure-rest-api-specs/blob/41270c6a8358273507993c6a630864ce0d93c732/specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2023-05-01/examples/ContainerGroupCreateConfidential.json#L31)